### PR TITLE
Mosip 24230 develop

### PR DESCRIPTION
**Pull request by AkshayChouguleac1**
Replaced AuthUtil with BioUtil

### CODE REVIEW

File: registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/exception/util/PlatformErrorMessages.java [hunk 1]
```diff
@@ -1416,7 +1416,10 @@ public enum PlatformErrorMessages {
 	RPR_FINALIZATION_STAGE_DRAFT_REQUEST_UNAVAILABLE(PlatformConstants.RPR_FINALIZATION_STAGE + "003",
 			"Draft request is unavaialble in id-repo draft repository."),
 	RPR_FINALIZATION_STAGE_API_RESOURCE_EXCEPTION(PlatformConstants.RPR_FINALIZATION_STAGE + "004",
-			"Not able to access the API resource")
+			"Not able to access the API resource"),
+	
+	INDIVIDUAL_BIOMETRIC_AUTHENTICATION_FAILED(PlatformConstants.RPR_BIOMETRIC_EXTRACTION_STAGE + "004",
+			"Bio authentication failed")
```
Comment (sowmya695) [inline registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/exception/util/PlatformErrorMessages.java:1422]: Please add RPR_BIOMETRIC_AUTHENTICATION_MODULE in constant as it is in bioauthentication stage

File: registration-processor/core-processor/registration-processor-biometric-authentication-stage/src/main/java/io/mosip/registration/processor/biometric/authentication/constants/BiometricAuthenticationConstants.java [hunk 2]
```diff
@@ -13,4 +13,5 @@ public class BiometricAuthenticationConstants {
 	public static final String CHILD = "CHILD";
 	public static final String NEW_PACKET_DESCRIPTION = "The packet is New and adult packet for registration id";
 	public static final String CHILD_PACKET_DESCRIPTION = "Skipped authentication as a child’s data was updated";
+	public static final String INDIVIDUAL_BIOMETRIC_NOT_IN_PACKET = "Individual biometric not in packet";
```
Comment (sowmya695) [inline registration-processor/core-processor/registration-processor-biometric-authentication-stage/src/main/java/io/mosip/registration/processor/biometric/authentication/constants/BiometricAuthenticationConstants.java]: Please remove this new error message if it is not used in code

File: registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/exception/util/PlatformSuccessMessages.java [hunk 3]
```diff
@@ -59,6 +59,9 @@ public enum PlatformSuccessMessages {
 	// Biometric Authentication Success Messages
 	RPR_PKR_BIOMETRIC_AUTHENTICATION(PlatformConstants.RPR_BIOMETRIC_AUTHENTICATION_MODULE + "000",
 			"Biometric Authentication Success"),
+	
+	INDIVIDUAL_BIOMETRIC_AUTHENTICATION_FAILED(PlatformConstants.RPR_BIOMETRIC_AUTHENTICATION_MODULE + "004",
+			"Bio authentication failed")
```
Comment (sowmya695) [inline registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/exception/util/PlatformSuccessMessages.java]: Please remove here and add in errormessages

File: registration-processor/core-processor/registration-processor-biometric-authentication-stage/src/main/java/io/mosip/registration/processor/biometric/authentication/constants/BiometricAuthenticationConstants.java [hunk 4]
```diff
@@ -12,5 +12,5 @@ public class BiometricAuthenticationConstants {
 	public static final String ADULT = "ADULT";
 	public static final String CHILD = "CHILD";
 	public static final String NEW_PACKET_DESCRIPTION = "The packet is New and adult packet for registration id";
-	public static final String CHILD_PACKET_DESCRIPTION = "Skipped authentication as a child’s data was updated";
+	public static final String CHILD_PACKET_DESCRIPTION = "The packet is child packet for registration id";
```
Comment (sowmya695) [inline registration-processor/core-processor/registration-processor-biometric-authentication-stage/src/main/java/io/mosip/registration/processor/biometric/authentication/constants/BiometricAuthenticationConstants.java]: Please revert the changes

Comment (AkshayChouguleac1) [reply registration-processor/core-processor/registration-processor-biometric-authentication-stage/src/main/java/io/mosip/registration/processor/biometric/authentication/constants/BiometricAuthenticationConstants.java]: Done

File: registration-processor/pom.xml [hunk 5]
```diff
@@ -129,7 +129,7 @@
 		<kernel.qrcodegenerator.zxing.version>1.2.1-SNAPSHOT</kernel.qrcodegenerator.zxing.version>
 		<kernel.logger.logback.version>1.2.1-SNAPSHOT</kernel.logger.logback.version>
 		<kernel.idvalidator.vid.version>1.2.1-SNAPSHOT</kernel.idvalidator.vid.version>
-		<object.store.khazana.version>1.2.0.1-SNAPSHOT</object.store.khazana.version>
+		<object.store.khazana.version>1.2.1-SNAPSHOT</object.store.khazana.version>
```
Comment (sowmya695) [inline registration-processor/pom.xml]: Please take changes from 'git pull upstream develop' to get latest changes from develop

Comment (AkshayChouguleac1) [reply registration-processor/pom.xml]: done

File: registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/exception/util/PlatformErrorMessages.java (hunk 1 above)
Comment (AkshayChouguleac1) [reply registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/exception/util/PlatformErrorMessages.java:1422]: added in constants
